docs/src/CodeMirror.css

```diff
@@ -0,0 +1,15 @@
+.cm-s-solarized.CodeMirror {
+  -moz-box-shadow: none;
+  -webkit-box-shadow: none;
+  box-shadow: none;
+}
+
+.highlight, .code-toggle.open {
+  background-color: #fdf6e3;
+}
+
+.cm-s-solarized .cm-comment { color: #93a1a1; }
+
+.CodeMirror {
+  height: auto;
```

Comment:
Ya, we fixed this on the last pull request that came in and I forgot to apply it here. I need to do this: https://github.com/react-bootstrap/react-bootstrap/blob/master/docs/assets/style.css#L71-L73. Thanks for the find, I'll update that.